src/test/resources/application.properties

```diff
@@ -1 +1,4 @@
-appconfig.injecttestdata=true
\ No newline at end of file
+appconfig.injecttestdata=true
+
+# LUCENE in ram only mode to prevent file lock error with concurrent execution
+spring.jpa.properties.hibernate.search.default.directory_provider=ram
```

Comment:
Das sollten wir generell für Lucene machen. Auch für die deployed Applikation. Wenn die DB in memory läuft, sollte der index auch in memory sein.

Reply:
Nein hat im testfall einen File Writelock error gegeben, deshalb hab ichs auf ram umgestellt

Reply:
Das mit dem Writelock ist mir bekannt. Meinte das wir "ram" nicht nur in `src/test/resources/application.properties` verwenden sondern auch in `src/main/resources/application.yml`. Ich hab die Option `ram` bis dato nicht gekannt.